sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -145,26 +145,28 @@ plot as shown below:
 Jupyter Notebooks
 =================
 
-At this point we should mention Jupyter (formerly IPython) notebooks.
-
-Jupyter notebooks are a fantastic tool for exploratory data analysis, and they
-are widely used across the "PyData" community. Bokeh integrates seamlessly with
-Jupyter notebooks. To view the above examples in a notebook, you would only change
-|output_file| to a call to |output_notebook| instead.
+Jupyter notebooks are a fantastic tool for exploratory data analysis, widely 
+used across the "PyData" community. They were previously known as IPython 
+notebooks. Bokeh integrates seamlessly with Jupyter notebooks. To view the 
+above examples in a notebook, you would only change |output_file| to a call 
+to |output_notebook| instead.
 
-A large number of static examples may be viewed directly online at the
+Many static examples may be viewed directly online at the
 `Bokeh NBViewer Gallery`_.
 
 The `Bokeh GitHub repository`_ also has a number of example notebooks in the
-``examples/plotting/notebook/`` directory. After cloning the repository,
-navigate there and run::
+``examples/howto`` directory. After cloning the repository, navigate there and run::
 
     ipython notebook
 
 You can open and interact with any of the notebooks listed in the index page
 that automatically opens up. In particular, you might check out the
-`interact_basic`_ and `interact_numba`_ examples that show how Bokeh can
-be used together with Jupyter interactive widgets.
+`examples/howto/notebook_comms/Jupyter Interactors.ipynb 
+<https://github.com/bokeh/bokeh/blob/master/examples/howto/notebook_comms/Jupyter%20Interactors.ipynb>`_ 
+and `examples/howto/notebook_comms/Numba Image Example.ipynb 
+<https://github.com/bokeh/bokeh/blob/master/examples/howto/notebook_comms/Numba%20Image%20Example.ipynb>`_ 
```

Comment:
It's unknown (to me at least) if `bokeh-tree` works inside the references at the bottom. I think it would, certainly worth trying. So, imagining:
```
In particular, you might check out the `Jupyter Interactors.ipynb`_ and ...

.. _Jupyter Interactors.ipynb: :bokeh-tree:`examples/howto/notebook_comms/Jupyter Interactors.ipynb`
```
If that doesn't work, then maybe something like just:

```
In particular, you might check out these examples:

:bokeh-tree:`examples/howto/notebook_comms/Jupyter Interactors.ipynb`
...
...

```